src/core/content/connector.ts

```diff
@@ -462,7 +464,7 @@ export default class BaseConnector {
 	 *
 	 * @returns The source URL
 	 */
-	public getOriginUrl: () => string | null = () => {
+	public getOriginUrl: () => string | null | undefined = () => {
```

Comment:
When would `document.location.href` return anything other than string?

Reply:
getOriginUrl is overrideable, and several connectors override it. document.location.href is always a string.

Reply:
Oh true! Need more :coffee: